flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/SemiJoinITCase.scala

```diff
@@ -470,7 +469,7 @@ class SemiJoinITCase(expectedJoinType: JoinType) extends BatchTestBase {
 }
 
 object SemiJoinITCase {
-  @Parameterized.Parameters(name = "{0}-{1}")
+  @Parameters(name = "{0}-{1}")
```

Comment:
It feels a bit strange, isn't this test class only one parameter?

Reply:
Yes. There is one parameter.

Reply:
Then we should fix it. Please try to give it a more appropriate name.

Reply:
Fixed.